src/components/Index/Environment.vue

```diff
@@ -99,7 +138,16 @@ export default {
           api_key: this.apiKey,
           bucketing: this.bucketing,
           timeout: this.timeout || 0,
-          polling_interval: this.pollingInterval || 0,
+          flagship_mode: this.flagshipMode || "api",
```

Comment:
_Breaking change_, we can keep sending bucketing to true if **bucketing** is selected otherwise false